### PR TITLE
Changelog 2.15.1

### PR DESCRIPTION
**Pull request by JoshLozensky**
# Changelog 2.15.1

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+2.15.1
+=========
+- Updated IdentityModel dependencies to latest version
```
Comment (jmprieur) [inline changelog.md:3]:
```suggestion
- Updated IdentityModel dependencies to Identity.Model.*.6.33.0 for all target frameworks others than .NET 8 rc1, for which Microsoft,Identity.Web leverages Identity.Model 7.0.2
```

**Review by jmprieur — APPROVED** [review]
LGTM
I provided a suggestion

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+2.15.1
+=========
+- Updated IdentityModel dependencies to Identity.Model.*.6.33.0 for all target frameworks others than .NET 8 rc1, for which Microsoft,Identity.Web leverages Identity.Model 7.0.2
```
Comment (jennyf19) [inline changelog.md:3]:
```suggestion
- Updated IdentityModel dependencies to Identity.Model.*.6.33.0 for all target frameworks other than .NET 8 rc1, for which Microsoft,Identity.Web leverages Identity.Model 7.0.2
```

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Pull request merged**